products/compute/api.yaml

```diff
@@ -2799,6 +2799,11 @@ objects:
             name: 'kmsKeyName'
             description: |
               The name of the encryption key that is stored in Google Cloud KMS.
+          - !ruby/object:Api::Type::String
```

Comment:
Sorry, I missed this earlier. This looks like a duplicate key though. Otherwise, I think it's looking good!

Reply:
fixed